process.cabal

```diff
@@ -66,18 +71,21 @@ library
     else
         c-sources:
             cbits/posix/runProcess.c
+            cbits/posix/bsd_closefrom.c
             cbits/posix/fork_exec.c
             cbits/posix/posix_spawn.c
             cbits/posix/find_executable.c
+        includes:
+            runProcess.h
+            cbits/posix/bsd_closefrom.h
+        install-includes:
+            runProcess.h
+            processFlags.h
+            cbits/posix/bsd_closefrom.h
```

Comment:
Do we really need to install this header?

Reply:
Probably not, I'll try to remove it once everything else is working smoothly.

Reply:
Removed now